zhaquirks/custom/telink.py

```diff
@@ -0,0 +1,209 @@
+    .removes(CustomUserInterfaceCluster.cluster_id, cluster_type=ClusterType.Client)
+    .adds(CustomUserInterfaceCluster)
+    .number(
+        CustomUserInterfaceCluster.AttributeDefs.temperature_offset.name,
+        CustomUserInterfaceCluster.cluster_id,
+        min_value=-327.67,
+        max_value=327.67,
+        step=0.01,
+        unit=UnitOfTemperature.CELSIUS,
+        translation_key="temperature_offset",
+        fallback_name="Temperature offset",
+        multiplier=0.01,
+        mode="box",
+    )
+    .number(
+        CustomUserInterfaceCluster.AttributeDefs.humidity_offset.name,
+        CustomUserInterfaceCluster.cluster_id,
+        min_value=-327.67,
+        max_value=327.67,
+        step=0.01,
+        unit=PERCENTAGE,
+        translation_key="humidity_offset",
+        fallback_name="Humidity offset",
+        multiplier=0.01,
+        mode="box",
+    )
+    .number(
+        CustomUserInterfaceCluster.AttributeDefs.comfort_temperature_min.name,
+        CustomUserInterfaceCluster.cluster_id,
+        min_value=-327.67,
+        max_value=327.67,
+        step=0.01,
+        unit=UnitOfTemperature.CELSIUS,
+        translation_key="comfort_temperature_min",
+        fallback_name="Comfort temperature min",
+        multiplier=0.01,
+        mode="box",
+    )
+    .number(
+        CustomUserInterfaceCluster.AttributeDefs.comfort_temperature_max.name,
+        CustomUserInterfaceCluster.cluster_id,
+        min_value=-327.67,
+        max_value=327.67,
+        step=0.01,
+        unit=UnitOfTemperature.CELSIUS,
+        translation_key="comfort_temperature_max",
+        fallback_name="Comfort temperature max",
+        multiplier=0.01,
+        mode="box",
+    )
+    .number(
+        CustomUserInterfaceCluster.AttributeDefs.comfort_humidity_min.name,
+        CustomUserInterfaceCluster.cluster_id,
+        min_value=0,
+        max_value=99,
+        step=1,
+        unit=PERCENTAGE,
+        translation_key="comfort_humidity_min",
+        fallback_name="Comfort humidity min",
+        multiplier=0.01,
+        mode="box",
+    )
+    .number(
+        CustomUserInterfaceCluster.AttributeDefs.comfort_humidity_max.name,
+        CustomUserInterfaceCluster.cluster_id,
+        min_value=0,
+        max_value=99,
+        step=1,
+        unit=PERCENTAGE,
+        translation_key="comfort_humidity_max",
+        fallback_name="Comfort humidity max",
+        multiplier=0.01,
+        mode="box",
+    )
+    .number(
+        CustomUserInterfaceCluster.AttributeDefs.measurement_interval.name,
+        CustomUserInterfaceCluster.cluster_id,
+        min_value=3,
+        max_value=255,
+        unit=UnitOfTime.SECONDS,
+        translation_key="measurement_interval",
+        fallback_name="Measurement interval",
+        mode="box",
+    )
+    .switch(
+        CustomUserInterfaceCluster.AttributeDefs.display.name,
+        CustomUserInterfaceCluster.cluster_id,
+        off_value=1,
+        on_value=0,
```

Comment:
Actually, I think we could also use `force_inverted` for this? Shouldn't really make a difference though.